lib/cartopy/io/shapereader.py

```diff
@@ -442,20 +443,24 @@ def acquire_all_resources(self, format_dict):
         # Iterate through all scales and levels and extract relevant files.
         modified_format_dict = dict(format_dict)
         scales = ('c', 'l', 'i', 'h', 'f')
-        levels = (1, 2, 3, 4)
+        levels = (1, 2, 3, 4, 5, 6)
         for scale, level in itertools.product(scales, levels):
-            modified_format_dict.update({'scale': scale, 'level': level})
-            target_path = self.target_path(modified_format_dict)
-            target_dir = os.path.dirname(target_path)
-            if not os.path.isdir(target_dir):
-                os.makedirs(target_dir)
-
-            for member_path in self.zip_file_contents(modified_format_dict):
-                ext = os.path.splitext(member_path)[1]
-                target = os.path.splitext(target_path)[0] + ext
-                member = zfh.getinfo(member_path.replace(os.sep, '/'))
-                with open(target, 'wb') as fh:
-                    fh.write(zfh.open(member).read())
+            # the combination c4 does not occur for some reason
+            if scale != "c" or level != 4:
```

Comment:
```suggestion
            if scale == "c" and level == 4:
                continue
```
I think it is more readable to be explicit about what you are skipping.

Reply:
Yes thanks I did that.